examples/src/python/example/tensorflow_custom_op/zero_out_op_interface.cpp

```diff
@@ -0,0 +1,46 @@
+// This code is from the guide in https://www.tensorflow.org/guide/extend/op.
+
+#include "tensorflow/core/framework/op.h"
+#include "tensorflow/core/framework/op_kernel.h"
+#include "tensorflow/core/framework/shape_inference.h"
+
+using namespace tensorflow;
+
+/* Describe the kernel. */
+/* TODO: put this in a separate file! */
```

Comment:
Is this TODO still accurate? Looks ok to me.

Reply:
This was a personal thing because the language in that tutorial is imo extremely unclear. This TODO will be removed.